superset-frontend/src/components/TableLoader.jsx

```diff
@@ -18,12 +18,13 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Table, Tr, Td } from 'reactable-arc';
+import memoize from 'lodash/memoize';
```

Comment:
I don't want to "rock the boat" on this PR, but this would only be the second instance of `lodash/memoize` in the codebase. The other one looks like it'd be a simple refactor to use the `useMemo` hook. 

Maybe this component is worth a revisit in a follow-up PR to address the following:
• Convert to a function component (removing UNSAFE method use)
• convert to `useMemo` to support the pattern
• convert to TypeScript while we're at it
• Make folders for TableLoader & TableView, which would contain the `index.tsx` file, a Storybook entry, and their respective `...test.(j|t)sx` file

Reply:
I totally agree. I converted it to functional component and typescript. Would you mind if I added a storybook and tests in a separate PR?

Reply:
@kgabryje Sounds good... if you can open an issue for that, it may be helpful in tracking the storybook/tests task.